pkg/trace/agent/run.go

```diff
@@ -39,9 +39,14 @@ func Run(ctx context.Context) {
 	}
 
 	cfg, err := config.Load(flags.ConfigPath)
+	//initializing logger before checking the error to ensure start-up errors are logged to the log file
+	//even if cfg is loaded with an error or does not exist, valid log file will still be created
```

Comment:
I think this explanation is needed in the PR, but not in the code. It only makes sense to initialise the logger at startup.